.github/workflows/on-push-build-all.yml

```
@@ -70,22 +75,85 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y build-essential
 
+    - name: Install LibMan CLI (for client-side libraries)
+      run: dotnet tool install -g Microsoft.Web.LibraryManager.Cli
+
+    - name: Build Blazor WebAssembly Frontend
+      run: |
+        echo "Building Blazor WebAssembly frontend..."
+        cd FilePiWeb
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)